src/analysis/Environments/Environments.models.ts

```diff
@@ -1,4 +1,4 @@
-import { App } from 'src/libs/ajax/leonardo/models/app-models';
+import { ListAppResponse } from 'src/libs/ajax/leonardo/models/app-models';
```

Comment:
I notice that `ListAppResponse` and `ListRuntimeItem` are similar objects with different naming conventions; I like `List...Item` more bc "Response" on its own suggests that the object holds the whole list. Name was pre existing so I'm not requesting a rename in this PR, but would you consider it for the followup?

Reply:
Yep, I have a follow-up note to fix this! Going to do it under the same ticket but separate PR to reduce noise here, something I've been told I need to work on :) .